Ms2Database/App.config

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file doing here?

Reply:
How do i remove this ? i think it got here because the rebase that ocurre this days.

Reply:
Remove the file